panoptes_aggregation/version/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '4.1.0'
+__version__ = '4.2.0'
```

Comment:
I might need to bump this to 5.0.0 because the output of the HDBSCAN reducer is different.